packages/@uppy/audio/src/audio-oscilloscope/index.ts

```diff
@@ -42,16 +42,16 @@ export default class AudioOscilloscope {
 
   private bufferLength: number
 
-  private dataArray: Uint8Array
+  private dataArray: Uint8Array | undefined
```

Comment:
Do you want to apply the change on the rest of those (or I can do it if you prefer)
```suggestion
  private dataArray?: Uint8Array
```